backend/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:latest
+
+WORKDIR /app
+
+COPY . .
+
+RUN go mod download
+RUN go build -o /kubecloud
+
+EXPOSE 8080
+CMD [ "/kubecloud" ]
```

Comment:
how it should take configs?

Reply:
- What i thought of that user will create its own config in backend directory and it will be copied with project inside container

Reply:
@AlaaElattar  No. that can't be the case that's why configs are either passed as env variables or as mounted config (you can't possibly build a separate image for every config change.

Reply:
- Fixed, now config file mounted from docker compose